hailolibero/hailolibero.py

```diff
@@ -87,13 +120,116 @@ async def check_auth(self):
         except ClientConnectorError as e:
             logging.error("Connection Error", str(e))
 
+    async def read_settings(self,dry_run=False):
```

Comment:
I think we can remove `dry_run` here as it doesn't do anything and read only is a safe operation